grafana-plugin/src/plugin/GrafanaPluginRootPage.tsx

```diff
@@ -62,7 +63,9 @@ import grafanaGlobalStyle from '!raw-loader!assets/style/grafanaGlobalStyles.css
 export const GrafanaPluginRootPage = (props: AppRootProps) => {
   return (
     <Provider store={rootStore}>
-      <PluginSetup InitializedComponent={Root} {...props} />
+      <DirectPagingProvider>
+        <PluginSetup InitializedComponent={Root} {...props} />
```

Comment:
This is another reason why we don't use Context, wrapping the whole App in a state provider that is being used in just one single place is not a good practice IMO.

MobX uses Context behind the scenes but it's just more easier to understand where the state is coming from compared to the context approach, which to me seems like a low level operation in React.